netbout-client/pom.xml

```diff
@@ -180,7 +180,13 @@
                         <artifactId>qulice-maven-plugin</artifactId>
                         <configuration>
                             <excludes>
-                                <exclude>findbugs:.*</exclude>
+                                <!--
+                                @todo #911:30min Fix findbugs errors in client
+                                 Some issues may not require fixing an should be
+                                 excluded. Filters in qulice 0.12 are broken, right
```

Comment:
@dmzaytsev I created and issue but after discussion with the team I closed it - due to the fact that issue I found is not reproducable in version 0.15.2 (Netbout uses 0.12)